### DISCUSSION TITLE
My repo has been removed from github.

### DISCUSSION THREAD

**Discussion by surya7765** [community/community#42695] [Repositories]
### Select Topic Area

Bug

### Body

Whenever I open my repo I get this message instead of my code. I don't have the code on my local desktop please help me recover my code.

This repository has been disabled
Access to this repository has been disabled by GitHub Staff as a result of a sensitive data removal request. You may contact [GitHub Support](https://github.com/contact) for more information or to appeal this decision.

**Comment by radar-blip** [comment]
Just contact support like the message says and explain the situation.
They will probably re-enable the repository but without any sensitive data included.

**Reply by surya7765** [reply]
Could you please provide me with link of message section?

**Reply by amalpotra** [reply]
There you go https://support.github.com/request
You can request for support here.

**Comment by jfullstackdev** [comment]
that's in the account category, if you will continue

https://support.github.com/contact/account
